### DISCUSSION TITLE
dangerouslySetInnerHTML on Fragment

### DISCUSSION THREAD

**Discussion by andriiradkevych** [preactjs/preact#4311] [General]
I have an string html , and I want to convert it to jsx . With `div` wrapper it is clear how to do it . But what about the case when I don't need any wrapper ?

```javascript
const htmlString = '<title>Head</title>';

<head>
 {htmlString} ?????
</head>
```

How can I convert string html to jsx in preact without any external library ? I use `Deno Fresh` , and here is described issue , which bring me here https://github.com/denoland/fresh/discussions/2360

**Comment by marvinhagemeister** [comment]
+1 This request has come up several times and it would make sense to add to Preact, at least on the server.

**Reply by nwalters512** [reply]
I'd love to see this feature added, and would be interested in contributing it. Do you think this would be approachable to someone who's brand new to the Preact codebase?
